dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -336,6 +342,25 @@ const decideBackgroundColour = (
 	return undefined;
 };
 
+const labelStyles = css`
```

Comment:
tiny - could this be:
```suggestion
const brandingLinkStyles = css`
```